cupy/_core/core.pyx

```diff
@@ -485,7 +485,32 @@ cdef class _ndarray_base:
         """
         return self.get().tolist()
 
-    # TODO(okuta): Implement itemset
+    
+    cpdef itemset(self, index, value):
```

Comment:
`itemset` should be callable with one argument (if size of self is 1). Use `*args` here.

Basically, this method must behave equivalently to the following NumPy code:
https://github.com/numpy/numpy/blob/ea0b1708b27e683201285da53c6afda24bc8ba74/numpy/core/src/multiarray/methods.c#L2985-L2987
https://github.com/numpy/numpy/blob/ea0b1708b27e683201285da53c6afda24bc8ba74/numpy/core/src/multiarray/methods.c#L768

Reply:
Thank you for checking my code.
I've forgotten the case of if size of self is 1.
I will fix this and also test code.